third_party/2and3/chardet/universaldetector.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, Union, AnyStr, Pattern
+from logging import Logger
+
+class UniversalDetector:
```

Comment:
Since this stub supports Python 2, it should derive from `object` explicitly to mark it as a new-style class:

```suggestion
class UniversalDetector(object):
```

Reply:
> # The following line will re-export UniversalDetector like the implementation does.
> # The "as ..." part is required for re-export in stubs.

So whatever classes we define inside the top-level chardet/ directory has to be exported like this in the `__init__.pyi`. Got it.